### ISSUE TITLE
support for influx v2 Tokens for authentication

### ISSUE THREAD

**Issue by rabarar** [alsuren/mijia-homie#315]
any chance there's support to authenticate to an influx v2 instance using a API Access Token?

**Comment by qwandor** [comment]
I don't think so currently, but you're welcome to send a PR to add support. Or see #182 for a workaround.

**Comment by rabarar** [comment]
Andrew,No worries. I used your homie-controller and wrote a telegraf socket input processor this evening…Best,RobOn Apr 14, 2023, at 6:11 PM, Andrew Walbran ***@***.***> wrote:﻿
I don't think so currently, but you're welcome to send a PR to add support. Or see #182 for a workaround.
